packages/api/methods.ts

```diff
@@ -230,3 +235,42 @@ export function holdBudgetForNextMonth(month, amount) {
 export function resetBudgetHold(month) {
   return send('api/budget-reset-hold', { month });
 }
+
+export function createSchedule(
+  schedule: Partial<ScheduleEntity>,
+  conditions: Partial<RuleConditionEntity>[],
+) {
+  return send('api/schedule-create', { schedule, conditions });
+}
+
+export function updateSchedule(
+  id: string,
+  conditions: Partial<RuleConditionEntity>[],
+  resetNextDate?: boolean,
+) {
+  return send('api/schedule-update', {
+    id,
+    conditions,
+    resetNextDate,
+  });
+}
+
+export function deleteSchedule(scheduleId: string) {
```

Comment:
Same comment on the other functions:
```suggestion
export function deleteSchedule(id: ScheduleEntity['id']) {
```